recipes/gumbo-parser/all/conanfile.py

```diff
@@ -0,0 +1,67 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class GumboParserConan(ConanFile):
+    name = "gumbo-parser"
+    description = "An HTML5 parsing library in pure C99"
+    topics = ("conan", "parser")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/google/gumbo-parser"
+    license = "Apache-2.0"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+        if self.settings.compiler == "Visual Studio":
+            raise ConanInvalidConfiguration("%s is not supported by Visual Studio" % self.name)
```

Comment:
I think MSVC is actually supported, because there is a visual studio solution.
Unless you don't know how to add MSVC support using environment variables, and `tools.vcvars`,
I would change the comment to say `This recipe does not support Visual Studio`

Reply:
Fixed comment. I prefer to omit MSVC support becase Gumbo has reported issues with C99 on MSVC and bundled project file does not support 64-bit MSVC build, it requires plenty of patches.

Reply:
Yeah, adding 64-bit support is not trivial